aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,31 +170,54 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are five types of capitalization. 
+1. Sentence case, where the first letter is capitalized and subsequent words begin with lowercase letters, with some exceptions, such as proper nouns.
+   (example: Storage volume) 
+2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
+   (example: "The Benefits of Cloud Storage Explained")
+3. PascalCase, where there are capital letters inside the word, almost always for branding purposes.
+    (example: GreenLake)
+4. All lower case, where every single word is lower case 
+    (example: "lower case exudes an approachable and casual vibe")
+5. All caps, where every letter is capitalized
+    (example: ALL LETTERS ARE CAPITALIZED)
+
+At HPE, almost all text should be written in sentence case. Never begin a sentence or phrase with a lower case letter because it is too casual for our brand. Do not write in all caps because it is equated with yelling and increases stress.
+
+Sentence case should be applied on:
+- Attribute labels
+- Column headings
+- Legends
+- Tabs
+- Menus
+- Section and subsection headings
 - Paragraphs of text
 - Tips
 - Notifications
 - Help
+- Buttons
+- Dialog titles
 
-Titles follow title capitalization rules:
-
-- Page/dialog titles
-- Section headings
-- Attribute labels
-- Table column headings
-- Legends
-- Tabs/Buttons
+Title case should only be applied on:
+- Page titles
+- Subsections/subheadings
```

Comment:
This seems counter to the direction we are moving on the site. We have been adjusting all subsection headings to use sentence case.

Reply:
Agree.  I thought we were changing everything, last time we spoke.

Reply:
@dahliaferrer noticing this content has not been moved. Shouldn't "subsections/subheadings" be under the sentence case section?

Reply:
Edit, I see you added it above but didn't remove it from here.